tools/tpm2_takeownership.c

```diff
@@ -111,12 +120,23 @@ static bool change_auth(TSS2_SYS_CONTEXT *sapi_context,
 static bool change_hierarchy_auth(TSS2_SYS_CONTEXT *sapi_context) {
 
     // change owner, endorsement and lockout auth.
-    return change_auth(sapi_context, &ctx.passwords.owner,
-                "Owner", TPM2_RH_OWNER)
-        && change_auth(sapi_context, &ctx.passwords.endorse,
-                "Endorsement", TPM2_RH_ENDORSEMENT)
-        && change_auth(sapi_context, &ctx.passwords.lockout,
-                "Lockout", TPM2_RH_LOCKOUT);
+    bool result = true;
```

Comment:
at some point we need to support handle as well so you can change loaded tpm objects auth value.